.github/workflows/test.yaml

```diff
@@ -233,26 +226,56 @@ jobs:
         if: steps.gen-id.outputs.RUN_TEST == 'true' && matrix.credential == 'aws'
         id: create-eks
         run: |
+          # Install eksctl
           curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
           sudo mv /tmp/eksctl /usr/local/bin
+
+          # Create EKS cluster
           eksctl create cluster \
             --name ${{ steps.gen-id.outputs.TEST_EKS_CLUSTER_NAME }} \
             --nodes-min 1 --nodes-max 2 --node-type t3.large \
             --zones ${{ env.AWS_ZONES }} \
             --managed \
             --region ${{ env.AWS_REGION }}
+
+          # Wait for the EKS cluster to be active
           while [[ "$(eksctl get cluster ${{ steps.gen-id.outputs.TEST_EKS_CLUSTER_NAME }} --region ${{ env.AWS_REGION }} -o json | jq -r .[0].Status)" != "ACTIVE" ]]; do
             echo "Waiting for EKS cluster to be created..."
             sleep 60
           done
+
+          # Update kubeconfig
           aws eks update-kubeconfig --region ${{ env.AWS_REGION }} --name ${{ steps.gen-id.outputs.TEST_EKS_CLUSTER_NAME }}
         timeout-minutes: 60
         continue-on-error: false
-      - name: Install Dapr
```

Comment:
Removing this step since Dapr is installed by default with Radius now https://github.com/radius-project/radius/issues/8032